tests/scale_test/notifier.py

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/env python3
+"""A script to send notifications to Lira.
+
+This script provides a set of actions to send notifications to a typical Lira instance
```

Comment:
What do you think about moving this documentation into a README in the `scale_test` directory?

Reply:
That is a good idea! I assume the readme in the `scale_test`  dir will not only cover the docs of this script but also cover docs for other scripts and instructions about how to run a scale test. So even we create a readme file in the dir, I still prefer keeping a minimal set of docs in this file, closer to the code. What do you think?

Reply:
Yes, that makes sense!